apps/landing/components/Terms/Hero.tsx

```diff
@@ -0,0 +1,431 @@
+"use client";
```

Comment:
We can should leave a todo here to convert this component to a server component once the new typography is ready.